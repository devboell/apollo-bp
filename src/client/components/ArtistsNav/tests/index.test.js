import React from 'react'
import artistsData from 'data/test/artists.json'

import { ArtistsNav } from '../index'


it('ArtistsNav component snapshot', () => {
  const wrapper = shallow(
    <ArtistsNav artists={artistsData} />,
  )
  expect(wrapper).toMatchSnapshot()
})
