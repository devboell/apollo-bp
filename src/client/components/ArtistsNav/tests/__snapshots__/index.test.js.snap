// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistsNav component snapshot 1`] = `
<styled.div>
  <styled.ul>
    <ListItem
      name="Heroshige"
      path="heroshige"
    />
    <ListItem
      name="Hokusai"
      path="hokusai"
    />
    <ListItem
      name="Utamaro"
      path="utamaro"
    />
    <ListItem
      name="Kunisada"
      path="kunisada"
    />
  </styled.ul>
</styled.div>
`;
